sycl/source/detail/graph_impl.hpp

```diff
@@ -467,6 +520,13 @@ class graph_impl {
   std::shared_ptr<node_impl>
   addSubgraphNodes(const std::list<std::shared_ptr<node_impl>> &NodeList);
 
+  /// Duplicates and Adds sub-graph nodes from an executable graph to this
+  /// graph.
+  /// @param NodeList List of nodes from sub-graph in schedule order.
+  /// @return An empty node is used to schedule dependencies on this sub-graph.
+  std::shared_ptr<node_impl>
+  DuplicateAndAddSubgraphNodes(const std::shared_ptr<graph_impl> &SubGraph);
```

Comment:
nitpicking  naming convention
```suggestion
  duplicateAndAddSubgraphNodes(const std::shared_ptr<graph_impl> &SubGraph);
```